### ISSUE TITLE
Qase reporting on rancher version and possibly downstream cluster versions

### ISSUE THREAD

**Issue by igomez06** [rancher/qa-tasks#1016]
Include in our Qase reporting the version of rancher used, as well as possibly the type/version of the downstream cluster.

**Comment by bmdepesa** [comment]
@rancher-max is this something that is included with the new parser?

**Comment by rancher-max** [comment]
I don't think so. It looks like this would require an update to the [TestRun Creation](https://github.com/rancher/tests/blob/main/validation/pipeline/qase/testrun/main.go) script.
